src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
+    
```

Comment:
Changing the resource file only trough the viewer, it added a lot of spaces here. Is that normal?